docs/Local.md

```diff
@@ -5,3 +5,37 @@ sccache defaults to using local disk storage. You can set the `SCCACHE_DIR` envi
 The default cache size is 10 gigabytes. To change this, set `SCCACHE_CACHE_SIZE`, for example `SCCACHE_CACHE_SIZE="1G"`.
 
 The local storage only supports a single sccache server at a time. Multiple concurrent servers will race and cause spurious build failures.
+
+## Direct mode
```

Comment:
I think the name for the mode is not descriptive of what it does, this is a bikeshedding issue though. `cache_cxx_macro_expansion` is imho clearer of what is happening.

Reply:
Yes, or something like preprocessor cache mode? I think we should at least keep references to direct mode somewhere since that's the terminology `ccache` uses, but I agree that the naming could be clearer.

Reply:
I agree, referencing _ccache's direct mode_